.circleci/config.yml

```diff
@@ -5,5 +5,5 @@ jobs:
       - image: starefossen/ruby-node:2-8
     steps:
       - checkout
-      - run: gem install bundler
-      - run: ./go ci_test
+      - run: bundle
```

Comment:
does the image have bundler pre-installed? or do we need to add `gem install bundler` as a step?

Reply:
Yep! (Tests pass.)